lineapy/plugins/jinja_templates/ray_dag_workflow.jinja

```diff
@@ -0,0 +1,28 @@
+import {{ MODULE_NAME }}
+import ray
+import pickle
+import pathlib
+
+ray.init(runtime_env = {{RAY_RUNTIME_ENV}}, storage = "{{RAY_STORAGE}}")
+
+{% for task_def in task_definitions %}
+{{ task_def }} 
+{% endfor %}
+
+# Specify argument values for your pipeline run.
+pipeline_arguments = {{ dag_params }}
+
+{% for task_name, task_def in tasks.items() %}
+{%- if task_def.return_vars|length > 0 %}
+{%- for var in task_def.return_vars %}{{ var }}{{ ',' if not loop.last else '' }}{%- endfor %}
+{%- else %}
+_
+{%- endif %} = task_{{task_name}}.bind(
+    {%- for var in task_def.user_input_variables %}pipeline_arguments["{{ var }}"]{{ ',' if not loop.last else '' }}{%- endfor %}{%- if task_def.loaded_input_variables|length > 0 and task_def.user_input_variables|length > 0 %},{%- endif %}{%- for var in task_def.loaded_input_variables %} {{var}} {{ ',' if not loop.last else '' }}{%- endfor %}
+)
+{% endfor %}
```

Comment:
This whole piece is repeated. use jinja's inheritence to write this to base and include in the different execution templates.

Reply:
Yup good catch, inheriting from ray_dag_base.jinija now and both templates are muuuch cleaner